css/grayscale.css

```diff
@@ -273,7 +283,7 @@ ul.nav.navbar-nav {
 .btn-circle:focus {
     outline: 0;
     color: #fff;
-    background: rgba(255,255,255,.1);
+    background: rgba(255, 255, 255, .1);
```

Comment:
nitpick, but general guideline in front end code is to have a space after a comma and/or semi colon